index.html

```diff
@@ -3,50 +3,199 @@
+      </p>
+      <ul class="tag-recent-work">
+        <li> CSS</li>
+        <li>HTML</li>
+        <li>Bootstrap</li>
+        <li>Ruby</li>
+      </ul>
+      <button type="button" class="button-see-project">
+        See project
+      </button>
+    </div>
+    <div class="container-works ">
+      <div class="card-content">
+        <h3 class="work-title">Profesional Art Printing Data</h3>
+        <p class="work-description">
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+          standard
+        </p>
+        <ul class="tag">
+          <li>Html</li>
+          <li>CSS</li>
+          <li>Ruby</li>
+        </ul>
+      </div>
+      <button type="button" class="see-project-card">
+        See Project
+      </button>
+      <div class="card-content">
+        <h3 class="work-title">Profesional Art Printing Data</h3>
+        <p class="work-description">
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+          standard
+        </p>
+        <ul class="tag">
+          <li>Html</li>
+          <li>CSS</li>
+          <li>Ruby</li>
+        </ul>
+      </div>
+      <button type="button" class="see-project-card">
+        See Project
+      </button>
+      <div class="card-content">
+        <h3 class="work-title">Profesional Art Printing Data</h3>
+        <p class="work-description">
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+          standard
+        </p>
+        <ul class="tag">
+          <li>Html</li>
+          <li>CSS</li>
+          <li>Ruby</li>
+        </ul>
+      </div>
+      <button type="button" class="see-project-card">
+        See Project
+      </button>
+      <div class="card-content">
+        <h3 class="work-title">Profesional Art Printing Data</h3>
+        <p class="work-description">
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+          standard
+        </p>
+        <ul class="tag">
+          <li>Html</li>
+          <li>CSS</li>
+          <li>Ruby</li>
+        </ul>
+      </div>
+      <button type="button" class="see-project-card">
+        See Project
+      </button>
+      <div class="card-content">
+        <h3 class="work-title">Profesional Art Printing Data</h3>
+        <p class="work-description">
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
+          standard
+        </p>
+        <ul class="tag">
+          <li>Html</li>
+          <li>CSS</li>
+          <li>Ruby</li>
+        </ul>
+      </div>
+      <button type="button" class="see-project-card">
+        See Project
+      </button>
+    </div>
+  </section>
+  <section class="about-me" id="about-me">
+    <div class="resume">
+      <h1>About me</h1>
```

Comment:
- [ ] H1 carries a title-like message for other devices such as screen readers, so it is professionally advisable to use h1 once on a webpage and you will agree with me that h1, which carries a title-like message should not be used for about me heading. I would advise you to use another hx for the satisfaction of this reason.